components/RPC-Connection.jsx

```diff
@@ -139,7 +139,7 @@ function applyFilter(value, filter, network, setReturnValue) {
     	break;
 		case "blocksToDays":
 			BlocksToDays(value, setReturnValue);
-			break;
+		break;
```

Comment:
```suggestion
    	break;
```
The original indentation is correct for the switch case